### DISCUSSION TITLE
Enhancing Transparency and Collaboration

### DISCUSSION THREAD

**Discussion by benjaco** [kartikk221/hyper-express#228] [General]
Hi Kartik,

First, thank you for your prompt responses to issues on serve-static. It's clear you're committed to the project, and your engagement with the community is invaluable. I've noticed that several issues, including questions about future updates, are being closed with references to changes planned for the next major release. However, with only the master branch visible, it's challenging for contributors to understand the project's direction or how they might contribute to these upcoming features.

Could we consider a way to increase visibility into the planned changes or the development process? For example, creating a development branch visible to the public, or even a roadmap or milestone list for the next major update, could significantly enhance community engagement and collaboration. This transparency would not only clarify the project's future but also open up opportunities for contributors like myself to offer more targeted support and contributions.

I'm here to help and very interested in contributing to the evolution of serve-static. Whether through code, testing, documentation, or any other area where you need support, I'd like to be involved.

Thank you for considering my suggestions. I'm looking forward to the possibility of contributing more actively to the project.

Best regards,

Benjamin

**Comment by kartikk221** [comment]
Hey Benjamin, you are 100% right in that most of my development up to this point has been done locally and not in branches since most work tended to be pushed out fairly often throughout updates hence didn't feel the need for a new branch each time.

I am currently busy with ongoing projects / development within my SaaS business hence haven't had any time to put my progress so far into a separate branch for others to contribute towards.

While I can't give an exact timeline, I will look to create a v7.x branch which will include the major changes to bring compatibility closer to Express.js!

Regarding contributions from the community, I think having a singular discussion which tracks all incompatibilities with Express.js would be a great start because so far all of my ideas and planning has been done within my own personal Notion / tracking software and I think it would be beneficial to track these for everyone to consider, contribute and critique publicly.

**Reply by smolinari** [reply]
So, will you transfer your ideas/ planning to a discussion? 

I would have thought an issue per incompatibility would be a better process, as it would ease management of the development. One could make a project and have a clear process with visibility of status in heading to or rather getting close to Express compatibility.  It will also ease collaboration. I've been trying to get attention to this project myself wherever I can, because as you know, I'm trying to make a NestJS adapter for Hyper Express. And, in the end, this "help from the outside" will help you (and everyone else involved) in the end with this project. :grin:  

Scott
